src/13.1/com/intellij/plugins/haxe/runner/debugger/HaxeDebugRunner.java

```diff
@@ -804,18 +809,35 @@ public StackFrame(Project project, Module module,
             (project, fileName,
              GlobalSearchScope.allScope(project));
         }
-        VirtualFile file = null;
+
+        java.util.Collection<VirtualFile> matches = new THashSet<VirtualFile>();
         if (!files.isEmpty()) {
           for (VirtualFile f : files) {
-            if (f.getPath() == mFileName) {
-              file = f;
-              break;
+            if (f.getPath().endsWith(mFileName)) {
+              matches.add(f);
             }
           }
-          if (file == null) {
-            file = files.iterator().next();
-          }
         }
+        if (matches.isEmpty()) {
+          // If we don't have a match yet, then walk the classpath looking for
+          // an appropriate file.
+          file = HaxelibClasspathUtils.findFileOnClasspath(module, mFileName);
+        } else if (matches.size() == 1) {
+          // Got one.  If it's a good file, keep it.  Otherwise, try to pick it
+          // out of the classpath.
+          VirtualFile possible = matches.iterator().next();
+          file = possible.isValid() ? possible : HaxelibClasspathUtils.findFileOnClasspath(module, possible.toString());
+        } else {
+          // Too many matches. Get the first that occurs on the classpath.
+          file = HaxelibClasspathUtils.findFirstFileOnClasspath(module, matches);
+        }
+
+        // Now, work around the fact that IDEA treats symlinks as separate files.
+        // XXX: This should be controlled via an UI option.
+        if (null != file && file.is(VFileProperty.SYMLINK)) {
```

Comment:
Yes, I would like to add a few checkboxes to the openfl/debugger pane.  Right now, I'm just putting comments in the code.